lib/reports/subscriber_lists_report.rb

```diff
@@ -0,0 +1,70 @@
+class Reports::SubscriberListsReport
+  attr_reader :start_of_day, :end_of_day
+
+  CSV_HEADERS = %i[title
+                   slug
+                   matching_criteria
+                   created_at
+                   individual_subscribers
+                   daily_subscribers
+                   weekly_subscribers
+                   individual_updates_for_date
+                   unsubscribed].freeze
+
+  def initialize(date)
+    @start_of_day = Time.zone.parse(date)
+    @end_of_day = Time.zone.parse(date).end_of_day
+  end
+
+  def call
+    non_empty_lists = SubscriberList.where(
```

Comment:
I don't think we should filter out empty lists (see card comment on 15th October). There's value in knowing a list exists, with no subscribers, at a point in time.

Reply:
Okay dokay, I've removed this filter so now we'll return all lists (dead or alive) that were created on/before the date given.